include/ddc/chunk.hpp

```diff
@@ -87,15 +97,17 @@ class Chunk<ElementType, DiscreteDomain<DDims...>>
     /** Constructs a new Chunk by move
      * @param other the Chunk to move
      */
-    Chunk(Chunk&& other) : base_type(std::move(other))
+    Chunk(Chunk&& other)
+        : base_type(std::move(static_cast<base_type&>(other)))
+        , m_allocator(std::move(other.m_allocator))
     {
         other.m_internal_mdspan = internal_mdspan_type();
     }
 
     ~Chunk()
     {
         if (this->m_internal_mdspan.data()) {
-            operator delete[](this->data(), std::align_val_t(64));
+            std::allocator_traits<Allocator>::deallocate(m_allocator, this->data(), this->size());
```

Comment:
This is a behaviour change, the default allocator doesn't provide aligned memory, does it?

Reply:
I guess it depends on the default allocator we choose.